test/tests/transfers/general_field/nearest_node/subdomain/main.i

```diff
@@ -110,7 +110,5 @@
     variable = from_sub_elem
     from_blocks = 1
     to_blocks = 1
-    # Value conflicts search triggers a greedy search which creates a diff
-    search_value_conflicts = false
```

Comment:
Does this mean there is an issue with the option still?

Reply:
There is an issue with nearest node with bounding boxes yes. If you pick a bounding box that is not too inflated you can get non-nearest results back

the diagnostics turns on the greedy search so it can give different results